library/clinical_rules.php

```diff
@@ -1731,7 +1727,7 @@ function database_check($patient_id, $filter, $interval = '', $dateTarget = '')
             if (exist_database_item($patient_id, $temp_df[1], $temp_df[2], $temp_df[3], $temp_df[4], $temp_df[5], $temp_df[6], $intervalType, $intervalValue, $dateTarget)) {
                 // Record the match
                 if ($cond_loop > 0) { // For multiple condition check
-                    $isMatch = $isMatch && true;
+                    $isMatch = $isMatch && 1;
```

Comment:
this is an odd line

Reply:
$isMatch is a conditional otherwise you're just setting to each other. still strange but, I just wanted to get the var types to agree. otherwise, beats me why it's there.